examples/label-small-scale-cluster-density.yaml

```diff
@@ -0,0 +1,65 @@
+tests :
+  - name : small-scale-cluster-density-v2
```

Comment:
Lets use payload instead of small scale for the TRT work. Might make running things easier when we implement.

Reply:
Example metadata https://gist.github.com/jtaleric/836bf99528a25b41e0b3f81bb7710ba1